crates/rattler_repodata_gateway/src/sparse/mod.rs

```diff
@@ -122,6 +122,7 @@ impl SparseRepoData {
         repo_data: impl IntoIterator<Item = &'a SparseRepoData>,
         package_names: impl IntoIterator<Item = PackageName>,
         patch_function: Option<fn(&mut PackageRecord)>,
+        strict_channel_priority: bool,
```

Comment:
It would also be good to add some documentation to this function on what `strict_channel_priority` means

Reply:
Fair, added it